src/emm/emm_cli.py

```diff
@@ -211,5 +225,25 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_content_width=100, allow_extra_args=True))
+@click.option(
+    "--op",
```

Comment:
Why include both `--op` and `--operation`? `--op` doesn't follow typically command line short options (which are usually a single character and a single dash). I think if we want to provide a short-hand option, it should be something like `-o`.